iroha/src/torii/mod.rs

```diff
@@ -138,6 +139,17 @@ impl<W: WorldTrait> Torii<W> {
     /// Can fail due to listening to network or if http server fails
     #[iroha_futures::telemetry_future]
     pub async fn start(self) -> iroha_error::Result<()> {
+        #[allow(clippy::unused_async)]
+        async fn recover(err: Rejection) -> Result<impl Reply, Rejection> {
```

Comment:
```suggestion
        async fn custom_http_error_code(err: Rejection) -> Result<impl Reply, Rejection> {
```

Reply:
Or something like that. Maybe also leave a comment on top of the function there.